test/functionalTest/cases/1492_limit_zero/limit_zero_types.test

```diff
@@ -0,0 +1,136 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+limit_zero_types
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. GET /v2/types with 'limit' == 0 and options=count and see no types and fiware-total-count: 0
+# 02. POST /v2/entities to create E1 entity with type T1
+# 03. POST /v2/entities to create E2 entity with type T2
+# 04. GET /v2/types with 'limit' == 0 and options=count and see no types and fiware-total-count: 2
+# 05. GET /v2/types with 'limit' == 0 and see no types
```

Comment:
In steps 01, 04, 05 you should get `[]` instead of empty payload.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/ff2b2251b1f0a0a1703c4e40b51d163f72bcede8